test/cypress/plugins/stubs.js

```diff
@@ -0,0 +1,409 @@
+'use strict'
+
+// NPM dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+const userFixtures = require('../../fixtures/user_fixtures')
+const gatewayAccountFixtures = require('../../fixtures/gateway_account_fixtures')
+const transactionDetailsFixtures = require('../../fixtures/transaction_fixtures')
+const cardFixtures = require('../../fixtures/card_fixtures')
+
+// Stub definitions added here should always use the fixture builders to generate request and response bodys.
+// The fixture builders used should be validated by also being used in the pact tests for the API endpoint, and they
+// should be written in a strict enough way the JSON they produce will adhere to a validated structure.
+module.exports = {
+  getUserSuccess: (opts = {}) => {
+    const aValidUserResponse = userFixtures.validPasswordAuthenticateResponse(opts).getPlain()
+    return [
+      {
+        predicates: [{
+          equals: {
+            method: 'GET',
+            path: '/v1/api/users/' + aValidUserResponse.external_id,
+            headers: {
+              'Accept': 'application/json'
+            }
+          }
+        }],
+        responses: [{
+          is: {
+            statusCode: 200,
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: aValidUserResponse
+          }
+        }]
+      }
+    ]
+  },
+  getGatewayAccountSuccess: (opts = {}) => {
+    const aValidGetGatewayAccountResponse = gatewayAccountFixtures.validGatewayAccountResponse(opts).getPlain()
+    return [
+      {
+        predicates: [{
+          equals: {
+            method: 'GET',
+            path: '/v1/frontend/accounts/' + opts.gateway_account_id,
+            headers: {
+              'Accept': 'application/json'
+            }
+          }
+        }],
+        responses: [{
+          is: {
+            statusCode: 200,
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: aValidGetGatewayAccountResponse
+          }
+        }]
+      }
+    ]
+  },
+  getGatewayAccountQueryParamsSuccess: (opts = {}) => {
+    const aValidGetGatewayAccountResponse = gatewayAccountFixtures.validGatewayAccountResponse(opts).getPlain()
+    return [
+      {
+        predicates: [{
+          equals: {
+            method: 'GET',
+            path: '/v1/frontend/accounts',
+            query: { accountIds: opts.gateway_account_id.toString() },
```

Comment:
Do we need **toString()** here

Reply:
Yes in this case it is needed, otherwise the query parameters don't match.